_workspace/02_design.md

```
@@ -1,428 +1,467 @@
+    .`as`(UserStatEntity::totalSucceededDays.toSnakeCase())         // total_succeeded_days
+    .count()
+    .`as`(UserStatEntity::totalAttendanceDays.toSnakeCase())        // total_attendance_days (신규)
 ```
 
-적용 클래스:
-- `MockKakaoProcessor : MockOAuthProcessor` (기존 `OAuthProcessor` 직접 구현 → 마커 인터페이스 구현으로 변경; 마커가 `OAuthProcessor`를 상속하므로 동작 동일)
-- `MockAppleProcessor : MockOAuthProcessor` (동상)
-
-`KakaoProcessor`/`AppleProcessor`는 그대로 `OAuthProcessor` 직접 구현 (마커 미적용).
+> `.count()`는 group된 도큐먼트 수를 센다. `userId`로 group + `userId+date` 유니크이므로 = 출석일 수. 출석일 0(도큐먼트 없는) 유저는 애초에 group되지 않아 자동 제외된다(분석 확인 완료).
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)